ext/tests/span_attribute/function_params_non_simple.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Check if function non-simple types can be passed as function params
 --SKIPIF--
-<?php if (PHP_VERSION_ID < 80100) die('skip requires PHP >= 8.1'); ?>
```

Comment:
I believe this test can work with all PHP versions, if you EXPECTF and use `%A` on that closure:

```
["three"]=>
  object(Closure)#2 (%d) {%A
  }
```

Reply:
Thank you! fixed